### DISCUSSION TITLE
Who won Best Director 🎥🏆in 2018?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#772] [Galaxy Trivia] [answered]
**Bonus:** For what movie?

**Comment by peterbaccaro** [comment]
The Best Director award for the year 2018 was won by **Guillermo del Toro** for his movie "**The Shape of Water**" at the 90th Academy Awards, which is also known as the Oscars.

**Comment by SANKARAMDAS** [comment, marked as answer]
"The Shape of Water” led the 90th Academy Awards on Sunday, winning best picture and best director (for Guillermo del Toro).

**Comment by ParanoidUser** [comment]
**Extra:** What color was his pocket square at the ceremony?

**Reply by peterbaccaro** [reply]
Guillermo del Toro wore a **teal** silk pocket square at the ceremony.

**Comment by SANKARAMDAS** [comment]
it's a kinda Teal-blue/black silk pocket square at the ceremony.

![2023-05-19](https://github.com/ParanoidUser/yolo/assets/31897843/f222e067-2e50-48e7-a7c5-7b275b306a96)
